thorlcr/activities/keyedjoin/thkeyedjoinslave.cpp

```diff
@@ -789,6 +790,7 @@ class CKeyedJoinSlave : public CSlaveActivity, public CThorDataLink, implements
 
                                 unsigned __int64 localFpos;
                                 unsigned files = owner.dataParts.ordinality();
+                                FPosTableEntry *fPosMap = owner.remoteDataFiles ? owner.fPosToNodeMap : owner.fPosToLocalPartMap;
```

Comment:
if remoteDataFiles is true doesn't fPosToNodeMap == fPosToLocalPartMap.  If so why this conditon?